go/testing/action.go

```diff
@@ -69,11 +74,19 @@ func RunTestCase(graph *Graph, driver Driver, test *pf.TestSpec) (scope string,
 
 		// Advance time to unblock the next PendingStat.
 		if nextReady != -1 {
-			if err := driver.Advance(nextReady); err != nil {
+			err := driver.Advance(nextReady)
+			if err == ErrAdvanceDisabled {
+				log.WithFields(log.Fields{
+					"delay": nextReady,
+					"task":  nextName,
+				}).Warn("sleeping until next task is ready.")
+			} else if err != nil {
 				return scope, fmt.Errorf("driver.Advance: %w", err)
+			} else {
+				// Advance completed
```

Comment:
nit: obvious comment is obvious :)

Also, prefer to [indent error flow](https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow).